Ruby/wc/file_content.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# class to provide file utility information.
+class FileContent
+  def initialize(file_content)
+    @file_content = file_content
+  end
+
+  def size
+    @file_content.bytesize
+  end
+
+  def lines_count
+    lines = @file_content.lines
+    lines.count
+  end
+
+  def word_count
+    @file_content.split(/\s+/).length
+  end
+
+  def char_count
+    @file_content.length
+  end
+end
```

Comment:
This class is perfectly fine - been a while with ruby and just wanted to get your thoughts.
Why not a `module`  instead? When I saw this my thought immediately went to a `module` coz it looks like you just need a place to collect all the utility functions. 🤔

Reply:
Actually, I haven't thought about it.
A module here makes more sense.

Reply:
updated to use a module now - https://github.com/aganapuram/CodingChallengesJC/pull/1/commits/bf412bff7976b2c488d8228367fc08f60855f4f7